manifests/params.pp

```diff
@@ -0,0 +1,22 @@
+#
+#
+# == Class: artifactory::params
+#
+# The params class of the artifactory module.
+#
+#
+class artifactory::params (
+    $artifactory_base       = '/var/opt/jfrog/artifactory',
```

Comment:
Isn't the typical pattern for this to override params defaults with params on the main class?  I noticed you've inherited the params class on some of the docker classes - I'm assuming that's what the params are here for, but why not just ensure the artifactory main class is loaded and get them from there?